core/src/main/java/com/google/instrumentation/stats/RpcConstants.java

```diff
@@ -40,86 +40,117 @@
   // RPC client {@link MeasurementDescriptor}s.
   public static final MeasurementDescriptor RPC_CLIENT_ERROR_COUNT =
       MeasurementDescriptor.create(
-          "/rpc/client/error_count",
+          "/grpc.io/client/error_count",
```

Comment:
Yeah, in our doc on naming conventions there are no leading slashes, I'll remove them now.